tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -232,6 +236,705 @@ def test_bobby_profiler_user_workflow_multi_batch_row_count_range_rule_and_colum
+                            },
+                        },
+                    },
+                ],
+            },
+        },
+    }
+
+    column_name: str = "fare_amount"
+
+    result = validator.expect_column_values_to_be_between(
+        column=column_name,
+        result_format="SUMMARY",
+        include_config=True,
+        auto=True,
+        profiler_config=custom_profiler_config,
+    )
+    assert result.success
+    assert result.expectation_config["kwargs"] == {
+        "column": "fare_amount",
+        "min_value": -52.0,
+        "strict_min": False,
+        "mostly": 1.0,
+        "result_format": "SUMMARY",
+        "include_config": True,
+        "auto": True,
+        "profiler_config": custom_profiler_config,
+        "batch_id": "90bb41c1fbd7c71c05dbc8695320af71",
+    }
+
+    result = validator.expect_column_values_to_be_between(
+        column=column_name,
+        min_value=0.0,
+        result_format="SUMMARY",
+        include_config=True,
+        auto=True,
+        profiler_config=custom_profiler_config,
+    )
+    assert not result.success
+    assert result.expectation_config["kwargs"] == {
+        "column": "fare_amount",
+        "min_value": 0.0,
+        "strict_min": False,
+        "mostly": 1.0,
+        "result_format": "SUMMARY",
+        "include_config": True,
+        "auto": True,
+        "profiler_config": custom_profiler_config,
+        "batch_id": "90bb41c1fbd7c71c05dbc8695320af71",
+    }
+
+    result = validator.expect_column_values_to_be_between(
+        column=column_name,
+        min_value=0.0,
+        mostly=8.75e-1,
+        result_format="SUMMARY",
+        include_config=True,
+        auto=True,
+        profiler_config=custom_profiler_config,
+    )
+    assert result.success
+    assert result.expectation_config["kwargs"] == {
+        "column": "fare_amount",
+        "min_value": 0.0,
+        "strict_min": False,
+        "mostly": 8.75e-1,
+        "result_format": "SUMMARY",
+        "include_config": True,
+        "auto": True,
+        "profiler_config": custom_profiler_config,
+        "batch_id": "90bb41c1fbd7c71c05dbc8695320af71",
+    }
+
+    with pytest.raises(AssertionError) as e:
+        # noinspection PyUnusedLocal
+        result = validator.expect_column_values_to_be_between(
+            column=column_name,
+            mostly=1.0,
+            result_format="SUMMARY",
+            include_config=True,
+            auto=False,
+            profiler_config=custom_profiler_config,
+        )
+    assert "min_value and max_value cannot both be None" in str(e.value)
+
+    # Use one batch (at index "1"), loaded by Validator as active_batch, for estimating Expectation argument values.
+    batch_request = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+        "data_connector_query": {"index": 1},
+    }
+
+    custom_profiler_config = {
```

Comment:
this should be a config type

Reply:
@donaldheppner Again -- this is a custom override, provided by users (who may find plain dictionary more convenient -- at least we have held this pattern for user-facing `batch_request` API).

Reply:
@donaldheppner This got done, based on our conversation.  Thanks!